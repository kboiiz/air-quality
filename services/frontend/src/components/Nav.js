import Cloud from "../img/cloud.png"
import { NavLink } from "react-router-dom"

const Nav = () => (
  <header className="nav-bar">
    <img className="logo" src={Cloud} alt="a logo shaped like a cloud"/>
    <nav className="navigation">
      <ul className="nav-links">
        <li><NavLink to="/">home</NavLink></li>
        <li><NavLink to="/contact">contact</NavLink></li>
        <li><NavLink to="/about">about</NavLink></li>
        <li><NavLink to="/profile">profile</NavLink></li>
      </ul>
    </nav>
  </header>
)


export default Nav