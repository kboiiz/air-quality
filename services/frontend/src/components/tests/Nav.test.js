import React from "react";
import { BrowserRouter } from 'react-router-dom'
import { render, screen } from '@testing-library/react'
import "@testing-library/jest-dom"
import userEvent from "@testing-library/user-event"
import Nav from "../Nav"


describe("navbar", () => {
  it("renders the navbar", () => {
    const { container } = render(<Nav />, {wrapper: BrowserRouter})
    expect(container).toMatchSnapshot()
  })
})