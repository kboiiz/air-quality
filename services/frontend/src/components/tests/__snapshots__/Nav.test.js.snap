// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar renders the navbar 1`] = `
<div>
  <header
    class="nav-bar"
  >
    <img
      class="logo"
      src="cloud.png"
    />
    <nav
      class="navigation"
    >
      <ul
        class="nav-links"
      >
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            home
          </a>
        </li>
        <li>
          <a
            class=""
            href="/contact"
          >
            contact
          </a>
        </li>
        <li>
          <a
            class=""
            href="/about"
          >
            about
          </a>
        </li>
        <li>
          <a
            class=""
            href="/profile"
          >
            profile
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
